build_defs/java.build_defs

```diff
@@ -539,6 +539,17 @@ def _java_toolchain(name : str, jdk_url : str|dict='', jdk:str='', visibility:li
             hashes = hashes,
         )
 
+    if jdk:
```

Comment:
This will always be true at this point

Reply:
Friday night syndrome 🤦 This simplifies some of the logic that follows. Thanks!